gnomock.go

```diff
@@ -252,7 +252,7 @@ func (g *g) stop(c *Container) error {
 		}
 	}
 
-	return nil
+	return cli.removeContainer(context.Background(), id)
```

Comment:
Please handle the "in progress" error gracefully (return without error), otherwise - fail with error. The error happens most likely due to the `AutoRemove` flag triggering container removal before the code reaches this line.

Reply:
I take care of it in https://github.com/orlangure/gnomock/pull/757/commits/a2b478c58f12c6efb0e3c2e9a18e4feb62bfe54a